docker/Dockerfile.release

```diff
@@ -16,7 +16,8 @@
 
 FROM openzipkin/zipkin-builder as built
 
-ARG source_branch
+ARG RELEASE_VERSION
```

Comment:
@anuraaga fyi I changed this as it is easier to test the dockerfile. the below line is a hack to ensure the variable is set.